Extension/package.json

```diff
@@ -43,6 +43,33 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "taskDefinitions": [
+      {
+        "type": "cppbuild",
+        "required": [
+          "command",
+          "label"
+        ],
+        "properties": {
+          "label": {
+            "type": "string",
+            "description": "Task's name."
+          },
+          "command": {
+            "type": "string",
+            "description": "The compiler path."
+          },
+          "args": {
+            "type": "array",
+            "description": "Additional build args."
```

Comment:
>build [](start = 39, length = 5)

Are these args used for compiling? If so, describe as "Additional compiler args" ?

Reply:
What was the resolution to this question?